groovy_magic.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 TWO SIGMA OPEN SOURCE, LLC
```

Comment:
not sure what this file is, should i go into beaker directory with other parts of python runtime?